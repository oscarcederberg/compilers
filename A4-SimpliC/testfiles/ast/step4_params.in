int a (int n){
    c = n + 10;
}