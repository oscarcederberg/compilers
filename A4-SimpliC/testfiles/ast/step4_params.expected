Program
  List
    FunctionDecl
      IdDecl ID="a"
      List
        VariableDecl
          IdDecl ID="n"
          Opt
      Block
        List
          Assign ID="c"
            Add
              VariableUse ID="n"
              Numeral NUMERAL="10"
